### PR TITLE
Bobbyg603/issue19

### LINKED ISSUE #19: Fix TODO in Reprocess E2E Test
We need to either move getCrashes to this repo so we can find a recent crash or post a crash with symbols each time.

### PR DESCRIPTION
**Pull request by bobbyg603**
Fixes #19

### CODE REVIEW

File: src/common/data/table-data/table-data-client/table-data-client.spec.ts [hunk 1]
```diff
@@ -0,0 +1,149 @@
+import { of } from 'rxjs';
```
Comment (bobbyg603) [inline src/common/data/table-data/table-data-client/table-data-client.spec.ts:1]: don't need this!

File: src/common/data/table-data/table-data-client/table-data-request.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { ApiDataFilterGroup } from '../../search/filter-group/api-data-filter-group';
+import { OrderFilter } from '../../search/order-filter/order-filter'; // TODO BG from index?
```
Comment (bobbyg603) [inline src/common/data/table-data/table-data-client/table-data-request.ts:2]: TODO BG add index.ts files to each folder to make imports cleaner

File: src/crashes/crashes-api-client/crashes-api-client.spec.ts [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    let pageData;
+    let rows;
+    let tableDataClient;
+    let tableDataClientResponse;
+    let Comments;
+    let IpAddress;
+
+    beforeEach(() => {
+        fakeFormData = createFakeFormData();
+        apiClientResponse = createFakeResponseBody(200);  // TODO BG
```
Comment (bobbyg603) [inline src/crashes/crashes-api-client/crashes-api-client.spec.ts:25]: why is there a todo here?

File: src/crashes/crashes-api-client/crashes-api-row.ts [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  public id: string;
+  public stackKey: string;
+  public stackKeyId: string;
+  public appName: string;
+  public appVersion: string;
+  public appDescription: string;
+  public userDescription: string;
+  public user: string;
+  public email: string;
+  public events: Array<Event>; // TODO BG is this correct?
```
Comment (bobbyg603) [inline src/crashes/crashes-api-client/crashes-api-row.ts:35]: Address TODO

**Review by daveplunkett — APPROVED** [review]

**Pull request merged**